ci/helm/preview/values.yaml

```diff
@@ -0,0 +1,77 @@
+expose:
+  Annotations:
+    helm.sh/hook: post-install,post-upgrade
+    helm.sh/hook-delete-policy: hook-succeeded
+  config:
+    # add tls configuration
+    domain: platform.dev.nuxeo.com
+    exposer: Ingress
+    http: true
+    tlsacme: true
+    tlsSecretName: platform-cluster-tls
+    urltemplate: '"{{.Service}}-{{.Namespace}}.{{.Domain}}"'
+
+cleanup:
+  Args:
+    - --cleanup
+  Annotations:
+    helm.sh/hook: pre-delete
+    helm.sh/hook-delete-policy: hook-succeeded
+
+nuxeo:
+  fullnameOverride: preview
+  nuxeo:
+    customParams: |-
+      elasticsearch.restClient.socketTimeoutMs=300000
+      elasticsearch.restClient.connectionTimeoutMs=300000
+    podAnnotations:
+      preview/scm.ref: $SCM_REF
+    podLabels:
+      branch: "$BRANCH_NAME"
+      team: platform
+      resource: pod
+      usage: preview
+    service:
+      annotations:
+        fabric8.io/ingress.annotations: |-
+          nginx.ingress.kubernetes.io/proxy-body-size: 0
+          nginx.ingress.kubernetes.io/server-snippet: ignore_invalid_headers off;
+      labels:
+        branch: "$BRANCH_NAME"
+        team: platform
+        resource: service
+        usage: preview
+    image:
+      repository: $DOCKER_REGISTRY/nuxeo/nuxeo
+      tag: $VERSION
+      pullPolicy: IfNotPresent
```

Comment:
This should go in the TEMP commit as it won't be needed once https://github.com/nuxeo/nuxeo-helm-chart/pull/8/files is merged and the nuxeo Helm chart is updated.

In fact, it would even not be needed in the TEMP commit by following https://github.com/nuxeo/nuxeo/pull/3856/commits/20d169b7cc06156e5161530bb6fa6609eed6860d#r401406736.

Reply:
It is also not needed regarding: 
```yaml
preview/scm.ref: $SCM_REF
```
This will make a change in the chart for each push and thus re-depoy the nuxe pod.

Reply:
removed the whole image value

Reply:
`repository` also? We may need it to point to docker registry.

Reply:
I think we need repository and version!
Otherwise the default values in the nuxeo Helm chart are:
```
repository: docker.packages.nuxeo.com/nuxeo/nuxeo
tag: latest
```
And we want to pull the image previously built and pushed to the internal Docker registry.
Or I missed something :smile:.

Reply:
Back in place!